use-timescale/vpc.md

```diff
@@ -48,69 +47,71 @@ In order to set up VPC peering you need the following permissions in your AWS ac
 
-The stages to create a secured connection between Timescale Services and your AWS infrastructure are:
+The stages to create a secured connection between $CLOUD_LONG services and your AWS infrastructure are:
 
-1. [Create a Peering VPC in Timescale][aws-vpc-setup-vpc]
+1. [Create a Peering VPC in $CONSOLE][aws-vpc-setup-vpc]
 1. [Complete the VPC connection in your AWS][aws-vpc-complete]
 1. [Set up security groups in your AWS][aws-vpc-security-groups]
 1. [Attach a Timescale Service to the Peering VPC][aws-vpc-connect-vpcs]
 
-### Create a Peering VPC in Timescale
+### Create a Peering VPC in $CONSOLE
 
 Create the VPC and the peering connection that enables you to securely route traffic 
-between Timescale and your own VPC in a logically isolated virtual network.
+between $CLOUD_LONG and your own VPC in a logically isolated virtual network.
 
 <Procedure>
 
-1.  In [Timescale Console > VPC][console-vpc], click `Create VPC`.
+1.  In [$CONSOLE > VPC][console-vpc], click `New VPC`.
+
+    The number of VPCs you can attach to your project depends on your [pricing plan][pricing-plans]. If you
+    need more VPCs either contact contact [support@timescale.com](mailto:support@timescale.com) or change 
+    your pricing plan in $CONSOLE. Each $CLOUD_LONG VPC can have as many peering connections as you need.
+
 1.  Choose your region and IP range, then click `Create VPC`. 
 
-    <img class="main-content__illustration"
-    src="https://assets.timescale.com/docs/images/tsc-vpc-create.png" 
-    alt="Create a new Timescale VPC"/>
+    ![Create a new VPC in $CLOUD_LONG](https://assets.timescale.com/docs/images/tsc-vpc-create.png)
 
 1.  For as many peering connections as you need:
 
     1. In the `VPC Peering` column, click `Add`.
     2. Enter information about your existing AWS VPC, then click `Add Connection`.
 
-        <img class="main-content__illustration"
-        src="https://assets.timescale.com/docs/images/tsc-vpc-add-peering.png"
-        alt="Create a new Timescale VPC"/>
+       ![Create a new $CLOUD_LONG $VPC](https://assets.timescale.com/docs/images/tsc-vpc-add-peering.png)
 
-Timescale sends a peering request to your AWS account so you can 
+$CLOUD_LONG sends a peering request to your AWS account so you can 
 [complete the VPC connection in AWS][aws-vpc-complete].
 </Procedure>
 
 
 ### Complete the VPC connection in AWS
  
-When you receive the Timescale peering request in AWS, edit your routing table to match 
-the `IP Range` and `CIDR block` between your AWS and Timescale VPCs.
+When you receive the $CLOUD_LONG peering request in AWS, edit your routing table to match 
+the `IP Range` and `CIDR block` between your AWS and $CLOUD_LONG VPCs.
+
+When you peer a VPC with multiple CIDRs, all CIDRs are added to the $CLOUD_LONG rules automatically.
+However, if you change your CIDRs, you have to recreate another peering connection to $CLOUD_LONG. 
```

Comment:
This point can generate confusion in customers. Please, can you rephrase something like: 
_However, after the peering has been completed, further changes in your VPC's CIDRs won't be detected automatically. If you need to refresh the CIDRs, please recreate the peering connection again._